src/msg.c

```diff
@@ -383,30 +380,7 @@ static bool property_changed(const char *name,
 	if (!session)
 		return false;
 
-	if (strcmp("config", name) == 0) {
-		if (session->config && strcmp(session->config, value) == 0)
-			goto done;
-
-		list = parser_config_to_list(value);
-		if (list == NULL) {
-			hal_log_error("[session %p] config: parse error!",
-				      session);
-			goto done;
-		}
-
-		if (parser_config_is_valid(list) != 0) {
-			hal_log_error("[session %p] config: invalid format!",
-				      session);
-			l_queue_destroy(list, l_free);
-			goto done;
```

Comment:
Remove `done` label to avoid the following warning:
```
src/msg.c: In function ‘property_changed’:
src/msg.c:396:1: error: label ‘done’ defined but not used [-Werror=unused-label]
 done:
 ^~~~
cc1: all warnings being treated as errors
Makefile:926: recipe for target 'src/src_knotd-msg.o' failed
make[1]: *** [src/src_knotd-msg.o] Error 1
Makefile:496: recipe for target 'all' failed
make: *** [all] Error 2
```

Reply:
Ok